examples/tutorials/advanced/grid_equalization.py

```diff
@@ -20,7 +20,7 @@
 grid = pygmt.datasets.load_earth_relief(
     resolution="03s", region=[-119.825, -119.4, 37.6, 37.825]
 )
-grid_dist = pygmt.grd2xyz(grid=grid, output_type="pandas")["elevation"]
+grid_dist = pygmt.grd2xyz(grid=grid, output_type="pandas")["earth_igpp"]
```

Comment:
I'm a little unsure if it's a good idea to use the `dataset.name` as the `grd2xyz` output column name. It's seems a little weird to have a column named `earth_igpp`.

Reply:
Yes, I was also not so happy with this when changing it. I personally prefer column names that contain the actual quantity (and in the best case, the unit), i.e., here `elevation` (`elevation_m`).

Reply:
@yvonnefroehlich Sorry for the slow response. I think we probably need to finalize PR  https://github.com/GenericMappingTools/pygmt/pull/2398 first because changes #2398 may affect our decisions.

Reply:
No worries. Actually, I think your response is here https://github.com/GenericMappingTools/pygmt/issues/2914#issuecomment-1999167297.
We can simply leave this PR as is and wait until #2398 is merged.

Reply:
Looks like we can try to continue here, as PR #2398 has been merged.

Reply:
Sorry, still need to wait for #3120.

Reply:
Yes sure, no worries. Try the find time to look at your comments in PR #2914.